devel/Dockerfile

```diff
@@ -57,6 +58,32 @@ RUN if [ "$TARGETPLATFORM" != "linux/arm/v7" ]; then \
         cd ../.. && rm -rf qemu; \
     fi
 
-ENV PATH="/opt/phoenix/utils:${PATH}"
+# install Jlink tools for 32-bit armv7 architecture
+RUN if [ "$TARGETPLATFORM" = "linux/arm/v7" ]; then \
```

Comment:
two more things:
 - these lines are mostly the same (the only difference is the filename downloaded), so maybe they could be unified (just  set some bash variable (eg. `JLINK_FILENAME`) under `if $TARGETPLATFORM` and do rest of the commands using this variable
 - I think the install package file is not being removed

Reply:
Applied in last force push, but when building I've encountered some issue with certificate and had to add `--no-check-certificate` to wget